src/actions/appActions.js

```diff
@@ -76,6 +77,10 @@ export const initAppAndRedirectAction = (appState: string, platform: string) =>
       const { collectiblesHistory = [] } = await storage.get('collectiblesHistory');
       dispatch({ type: SET_COLLECTIBLES_TRANSACTION_HISTORY, payload: collectiblesHistory });
 
+      const { offlineQueue = [] } = await storage.get('offlineQueue');
+      dispatch({ type: UPDATE_OFFLINE_QUEUE, payload: offlineQueue });
+      dispatch({ type: START_OFFLINE_QUEUE });
```

Comment:
Maybe it's better to rename to `PROCESS_OFFLINE_QUEUE`?

Reply:
It is ok to have it as "start" for this action, some queues use "digest", "process" it's up to you.